### PR TITLE
fix conversion webhook in hosted mode

### PR DESCRIPTION
**Pull request by ldpliu**
Signed-off-by: ldpliu <daliu@redhat.com>
https://github.com/open-cluster-management-io/OCM/issues/67

### CODE REVIEW

File: pkg/helpers/sa_syncer.go [hunk 1]
```diff
@@ -155,7 +155,7 @@ func applyKubeconfigSecret(ctx context.Context, templateKubeconfig *rest.Config,
 		},
 		AuthInfos: map[string]*clientcmdapi.AuthInfo{
 			"user": {
-				TokenFile: "token",
+				Token: string(token),
```
Comment (qiujian16) [inline pkg/helpers/sa_syncer.go]: why make this change?

Comment (xuezhaojun) [reply pkg/helpers/sa_syncer.go]: Because the path is not right.

It could be fixed in this way:
```
TokenFile: "/var/run/secrets/hub/token"
```

**Review by qiujian16 — COMMENTED** [review]
/approve

**Pull request merged**